frontend/src/main/java/starter/project/frontend/AppConfig.java

```diff
@@ -0,0 +1,18 @@
+package starter.project.frontend;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import starter.project.domain.Car;
+
+import java.util.Map;
+
+@Configuration
+public class AppConfig {
+
+    @Bean
+    public Map<Integer, Car> map()
+    {
+        return Map.of();
```

Comment:
Have you run the endpoint? As this map is immutable and will error out when running it. You just need to change it to a normal hashmap.